<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToTxtDlg</class>
 <widget class="QDialog" name="ExportToTxtDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to text file</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qHostMonitor.qrc">
    <normaloff>:/img/export.png</normaloff>:/img/export.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbtnAllTests">
        <property name="text">
         <string>Export all tests</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnFolderWithSubs">
        <property name="text">
         <string>Export current folder (including subfolders)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnFolderOnly">
        <property name="text">
         <string>Export current folder only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnTestsSelected">
        <property name="text">
         <string>Export selected test(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbxIncludeFolder">
        <property name="text">
         <string>Include folder information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbxIncludeLinks">
        <property name="text">
         <string>Include links information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblTargetFile">
          <property name="text">
           <string>Target file</string>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
          <property name="buddy">
           <cstring>editTargetFile</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editTargetFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnTargerFile">
          <property name="minimumSize">
           <size>
            <width>27</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>27</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qHostMonitor.qrc">
            <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qHostMonitor.qrc"/>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportToTxtDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportToTxtDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
